src/main/java/org/apache/accumulo/testing/continuous/ContinuousIngest.java

```diff
@@ -191,14 +205,36 @@ public static void main(String[] args) throws Exception {
           pauseCheck(testProps, r);
         }
 
-        // create one big linked list, this makes all of the first inserts point to something
-        for (int index = 0; index < flushInterval - 1; index++) {
-          Mutation m = genMutation(firstRows[index], firstColFams[index], firstColQuals[index], cv,
-              ingestInstanceId, count, genRow(prevRows[index + 1]), checksum);
-          count++;
-          bw.addMutation(m);
+        // random chance that the entries will be deleted
+        boolean delete = r.nextFloat() < deleteProbability;
+
+        // if the previously written entries are scheduled to be deleted
+        if (delete) {
+          log.info("Deleting last portion of written entries");
+          // add delete mutations in the reverse order in which they were written
+          for (int depth = nodeMap.length - 1; depth >= 0; depth--) {
+            for (int index = nodeMap[0].length - 1; index >= 0; index--) {
+              MutationInfo currentNode = nodeMap[depth][index];
+              Mutation m = new Mutation(genRow(currentNode.row));
+              m.putDelete(genCol(currentNode.cf), genCol(currentNode.cq));
+              count--;
```

Comment:
Not sure we should decrement this.   Makes it hard to understand what it is happening.  I think a separate delete counter may be better, then we can see number of inserts and deletes that have happened.

Reply:
Addressed in 5e0fcf8. I have a few improvements in mind that are unrelated to adding deletes. I plan to make a separate PR after this one gets merged and will add a delete counter there. For now I have removed the decrementing the counter.